src/gwt/src/org/rstudio/studio/client/rsconnect/ui/NewPositCloudPage.java

```diff
@@ -29,7 +28,7 @@
 {
    public NewPositCloudPage()
    {
-      super(RSConnect.CLOUD_SERVICE_NAME,
+      super("Posit Cloud",
```

Comment:
We need to hardcode this because RSConnect.CLOUD_SERVICE_NAME is `rstudio.cloud`, and will eventually be changed to `posit.cloud` but we want `Posit Cloud` with capitalization and space as a display name